swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -444,6 +446,51 @@ public void setSelectedOption(final O _option) {
 			setSelectedItem(item);
 		}
 	}
+	
+	/////////////////////////////////////////////////////////////////////////
+	//
+	// Deal with non-standard GlazedList UP/DOWN arrow handling
+	//
+	
+    /**
+     * When dealing with GlazedLists (1.11) restore expected combo behavior.
+     *
+     * Per GL JavaDoc:
+     * https://javadoc.io/doc/com.glazedlists/glazedlists/latest/ca/odell/glazedlists/swing/AutoCompleteSupport.html
+     *<p>
+     *  4. typing the up arrow key when the popup is visible and the selected element is the first element causes the autocompletion to be cleared and the popup's selection to be removed.
+     *  6. typing the down arrow key when the popup is visible and the selected element is the last element causes the autocompletion to be cleared and the popup's selection to be removed
+     *<p>
+     * We want to restore the normal JComboBox behavior of not going past the first or last item. This would be the ideal case, matching JComboBox.
+     * If GlazedLists ever changes the arrow key behavior, this can be removed.
+     */
+	protected void glazedListArrowHandler() {
+		// TODO: if not dealing with GlazedList then we can just return.
+		
+        // ADD KEY LISTENER - INTERCEPTING KEYPRESSED APPEARS TO BLOCK GL
+        getEditor().getEditorComponent().addKeyListener(new KeyAdapter() {
```

Comment:
glazed promises to install an editor. Other situations might get NPE.

Reply:
I agree. About to push a commit that calls arrow handler only for GL version.